pom.xml

```diff
@@ -86,6 +86,7 @@
         <bcprov-ext-jdk18on.version>1.78</bcprov-ext-jdk18on.version><!-- TODO: Remove this. Required as there is no 1.78.1 version released -->
         <brotli.version>0.1.2</brotli.version><!-- @sync org.apache.httpcomponents.client5:httpclient5-parent:${httpclient5.version} prop:brotli.version -->
         <caffeine.version>3.1.5</caffeine.version><!-- @sync io.quarkus:quarkus-bom:${quarkus.version} dep:com.github.ben-manes.caffeine:caffeine -->
+        <certificate.generator.version>0.5.0</certificate.generator.version>
```

Comment:
Nitpick - Can you move this with the other test dependency properties:

https://github.com/apache/camel-quarkus/blob/7eebe0453ff401cc083f838aa247ed285688d800/pom.xml#L171-L180

Reply:
sure, I missed the separation for test versions

Reply:
moved